### PR TITLE
⚡ optimize image sizes and 🧹 cleanup css

### PR DESCRIPTION
**Pull request by bmiddha**
- Change image sizes to match dom element size.
- Cleanup unneeded css files and rules with infima utility classes.
- Update author links for @ChristianBingman

### CODE REVIEW

File: blog/2021-05-02-new-blog.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: New blog 👋
 author: Bharat Middha
 author_title: President 2019-2021 @ ACM@UIC
 author_url: https://github.com/bmiddha
-author_image_url: https://avatars.githubusercontent.com/u/5100938?s=200&v=4
+author_image_url: https://avatars.githubusercontent.com/u/5100938?s=50&v=4
```
Comment (clee231) [inline blog/2021-05-02-new-blog.md:7]: I see the computed size of this is around ~50px (at least on my display). For Retina/HiDPI displays, wouldn't this show up a little blurry?

**Review by clee231 — CHANGES_REQUESTED** [review]

File: src/components/HomepageProfiles/index.ts [hunk 2]
```diff
@@ -1 +1,2 @@
 export * from "./HomepageProfiles";
+export { HomepageProfiles as default } from "./HomepageProfiles";
```
[the remark this answers is not in the document]
Comment (bmiddha) [reply src/components/HomepageProfiles/index.ts:2]: We don't need it but, I like adding both in the index files. passing a default exports over and the others as well.

File: blog/2021-05-02-new-blog.md (hunk 1 above)
Comment (bmiddha) [reply blog/2021-05-02-new-blog.md:7]: I don't think this is relevant for avatars. If we had hero images for a blog post or another content relevant image, I'd agree with going a little higher res.
This particular image is 19kB when `s=200` and 2.7kB when`s=50`. That's a considerable difference.

Comment (bmiddha) [reply blog/2021-05-02-new-blog.md:7]: The rendered image size is small enough that I could not tell the difference. I tested on a couple screens ranging from 130 to 270 PPI.

**Review by clee231 — APPROVED** [review]
Sounds good to me! :+1:

**Pull request merged**